src/Microsoft.AspNetCore.SignalR.Redis/RedisHubLifetimeManager.cs

```diff
@@ -187,7 +188,7 @@ public override Task OnConnectedAsync(HubConnectionContext connection)
 
             if (connection.User.Identity.IsAuthenticated)
             {
-                var userChannel = _channelNamePrefix + ".user." + connection.User.Identity.Name;
+                var userChannel = _channelNamePrefix + ".user." + connection.User.FindFirst(ClaimTypes.NameIdentifier)?.Value;
```

Comment:
@BrennanConroy Ok, let's assume it's possible case, that user authenticated, but still have no  `NameIdentifier` claim. What behavior you expect to see there. Just do nothing or throwing `InvalidOperationException` ?